lib/scope.js

```diff
@@ -110,15 +114,11 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
     var in_destructuring = null;
     var in_export;
     var tw = new TreeWalker(function(node, descend){
-        var create_a_block_scope =
-            (options.screw_ie8 && node instanceof AST_Catch) ||
```

Comment:
I guess these tests are correct? Or at least, that's how they result on master:

```js
do_screw_try_catch: {
    options = { screw_ie8: true };
    mangle = { screw_ie8: true };
    beautify = { screw_ie8: true };
    input: {
        bad = function(e){
            return function(error){
                try{
                    e()
                } catch(e) {
                    error(e)
                }
            }
        };
    }
    expect_exact: "bad=function(n){return function(t){try{n()}catch(n){t(n)}}};"
}

dont_Screw_try_catch: {
    options = { screw_ie8: false };
    mangle = { screw_ie8: false };
    beautify = { screw_ie8: false };
    input: {
        bad = function(e){
            return function(error){
                try{
                    e()
                } catch(e) {
                    error(e)
                }
            }
        };
    }
    expect_exact: "bad=function(n){return function(t){try{n()}catch(e){t(e)}}};"
}
```

As expected, they are troubling for harmony at this point.